docs/2.guide/2.directory-structure/1.pages.md

```diff
@@ -260,17 +260,17 @@ definePageMeta({
 
 ### Special Metadata
 
-Of course, you are welcome to define metadata for your own use throughout your app. But some metadata defined with `definePageMeta` has a particular purpose:
+Of course, you are welcome to define metadata throughout your app. But some metadata defined with `definePageMeta` has a particular purpose:
 
 #### `alias`
 
-You can define page aliases. They allow you to access the same page from different paths. It can be either a string or an array of strings as defined [here](https://router.vuejs.org/guide/essentials/redirect-and-alias.html#alias) on vue-router documentation.
+You can define page aliases. They allow you to access the same page from different paths. It can be a string or an array of strings as defined [here](https://router.vuejs.org/guide/essentials/redirect-and-alias.html#alias) on vue-router documentation.
```

Comment:
```suggestion
You can define page aliases. They allow you to access the same page from different paths. It can be a string or an array of strings as defined in the [Vue Router docs](https://router.vuejs.org/guide/essentials/redirect-and-alias.html#alias).
```